src/app/Job-Service/job.service.spec.ts

```diff
@@ -1,12 +1,20 @@
 import { TestBed } from '@angular/core/testing';
-
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 import { JobService } from './job.service';
 
 describe('JobService', () => {
-  beforeEach(() => TestBed.configureTestingModule({}));
+  let jobService: JobService; // added for testing
 
-  it('should be created', () => {
-    const service: JobService = TestBed.get(JobService);
-    expect(service).toBeTruthy();
+  beforeEach(() =>{
+    TestBed.configureTestingModule({
+    imports: [HttpClientTestingModule],
+    providers: [JobService] 
   });
+  jobService = TestBed.get(JobService); //added for testing
 });
+
```

Comment:
This can perhaps be in a different Pull Request, but this service should have more tests.

Reply:
We have set up a new branch for tests outside the scope of the components/services we directly touched with the add skill functionality. We will make a separate pull request when those tests are ready.